content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,179 @@
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in *Packs* from the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - :guilabel:`Starter Pack`: 10 credits
+   - :guilabel:`Standard Pack`: 100 credits
+   - :guilabel:`Advanced Pack`: 500 credits
+   - :guilabel:`Expert Pack`: 1,000 credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits. Users can check the current balance of credits for each service, and manually
+purchase more credits by navigating to the :menuselection:`Settings app --> Search IAP --> View My
+Services`.
+
+Doing so reveals an :guilabel:`IAP Account` page, listing the various IAP services in the database.
+From here, click an IAP service to open its :guilabel:`Account Information` page, where additional
+credits can be purchased.
+
+Manually buy credits
+~~~~~~~~~~~~~~~~~~~~
+
+To manually buy credits in Odoo, follow these steps:
+
+#. Go to the *Settings* application.
+#. Type `IAP` in the search bar at the top.
+#. Click :guilabel:`View My Services`.
+
+   .. image:: in_app_purchase/view-services.png
+      :align: center
+      :alt: The Settings app showing the Odoo IAP heading and View My Services button.
+
+#. The available IAP accounts appear in a list view. Click on the desired IAP account to view the
+   :guilabel:`Account Information` page.
+#. On the :guilabel:`Account Information` page, click the :guilabel:`Buy Credit` button. A
+   :guilabel:`Buy Credits for (IAP Account)` page loads in a new tab.
+
+   .. image:: in_app_purchase/account-info.png
+      :align: center
+      :alt: The Account Information page for an IAP service showing the Buy Credit button.
+
+#. From here, click :guilabel:`Buy` on the desired pack of credits, then follow the prompts to enter
+   payment details, and confirm the order.
+#. Once the transaction is complete, the credits are available for use in the database.
+
+   .. image:: in_app_purchase/buy-pack.png
+      :align: center
+      :alt: The SMS service page on IAP.Odoo.com with four packs of credits available for purchase.
+
+.. _in_app_purchase/low-credits:
+
+Low-credit notification
+~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to be notified when credits are low, in order to avoid running out of credits while
+using an IAP service. To do that, follow this process:
```

Comment:
```suggestion
It is possible to be notified when credits are low, in order to avoid running out of credits, while
using an IAP service. To do that, follow this process:
```